Tests/MapboxSearchIntegrationTests/search-box/SearchBox_SearchEngineIntegrationTests.swift

```diff
@@ -158,4 +158,29 @@ class SearchBox_SearchEngineIntegrationTests: MockServerIntegrationTestCase<Sear
         wait(for: [successExpectation], timeout: 10)
         XCTAssertFalse(searchEngine.suggestions.isEmpty)
     }
+
+    func testRetrieveMapboxIDQuery() throws {
+        try server.setResponse(.retrieveMapboxID)
+
+        let successExpectation = delegate.successExpectation
+        searchEngine.retrieve(mapboxID: "dXJuOm1ieHBvaTo0ZTg2ZWFkNS1jOWMwLTQ3OWEtOTA5Mi1kMDVlNDQ3NDdlODk")
+        wait(for: [successExpectation], timeout: 10)
+
+        let result = try XCTUnwrap(delegate.resolvedResult)
+        let metadata = try XCTUnwrap(result.metadata)
+
+        XCTAssertNil(result.accuracy)
+        XCTAssertNil(result.distance)
+        XCTAssertEqual(result.categories, ["museum", "tourist attraction"])
+        XCTAssertNotNil(result.routablePoints?.first)
+
+        XCTAssertNil(metadata.children)
+        XCTAssertNotNil(metadata.primaryImage)
```

Comment:
Do I understand correctly that the response is provided from the local file `Tests/MockData/search-box/search-box-retrieve-mapbox-id.json`? Why not test the exact values then? It is an integrating test, it would be more useful if it would help to catch the problems in filling specific metadata fields.